components/groups/modals/voteDetailsModal.tsx

```diff
@@ -203,7 +203,10 @@ function VoteDetailsModal({
     >
       <div className="fixed inset-0 bg-black/30" aria-hidden="true" />
 
-      <Dialog.Panel className="relative flex flex-col items-center justify-center w-full h-full">
+      <Dialog.Panel
+        className="relative flex flex-col items-center justify-center w-full h-full"
+        onClick={onClose}
```

Comment:
That closes the dialog when clicking the dialog's content, no?

Reply:
No. Try it!